src/components/common/Fields.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Field extends Component {
+  render() {
+    return (
+            <div className="form-group">
+                <label htmlFor={this.props.htmlFor} className='form-label'>{this.props.label}</label>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['htmlFor' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)